cmd/aasp/main.go

```diff
@@ -59,10 +58,7 @@ var (
 )
 
 const (
-	aasp                       = "aasp"
-	AZURE_CLIENT_ID            = "AZURE_CLIENT_ID"
-	AZURE_TENANT_ID            = "AZURE_TENANT_ID"
-	AZURE_FEDERATED_TOKEN_FILE = "AZURE_FEDERATED_TOKEN_FILE"
+	aasp = "aasp"
```

Comment:
Why is this lower case? General standards for constants is to have the names be upper case.

Reply:
done